english/friend-scream/follow.md

```diff
@@ -0,0 +1 @@
+You followed the sound to the ground floor and you realize, the noise is just another recording on a loop. WTF! You placed yourself back in danger for nothing. Where did they get that recording from? Who are 'they'? These are all questions you ask yourself. Just then, you see a figure in the distance. It's a...It's a...another Marshmallow man! This time he's more than just hands. He is HUGE! and he has sharp teeth! you realize that you've noticed these things about him because he is now 10 feet away from you! What do you do?!....
```

Comment:
It looks like this file has been added to a directory that already contained a "page" of the story.  Please create a new directory for each new page to help keep the repository organized.  See the README of this repository for more information.